### PR TITLE
[TIMOB-24908] Prevent building with incompatible Windows SDK 10.0.15063.0

### PR DESCRIPTION
**Pull request by garymathews**
- Titanium Windows is currently incapable of building with `10.0.15063.0` which should only be used with Visual Studio 2017

###### TEST CASE
- Install `10.0.15063.0` (https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk)
- Attempt to build a Titanium Windows project

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-24908)

### CODE REVIEW

File: cli/commands/_build/generate.js [hunk 1]
```diff
@@ -106,8 +106,31 @@ function generateI18N(next) {
...
+		this.windowsInfo.windowsphone['10.0'].sdks &&
+		this.windowsInfo.windowsphone['10.0'].sdks.length > 0) {
+		
+		// set target platform version to highest version available
+		// that is below 10.0.15063, as current Titanium Windows is incompatible
+		var sdks = this.windowsInfo.windowsphone['10.0'].sdks;
+		for (var i = sdks.length - 1; i > 0; i--) {
+			var sdk = sdks[i],
+				version = sdk.split('.');
+			if (parseInt(version[2]) < 15063) {
```
Comment (ewanharris) [inline cli/commands/_build/generate.js]: I think this is something that should reside in the package.json similar to [iOS](https://github.com/appcelerator/titanium_mobile/blob/master/iphone/package.json#L22) and [Android](https://github.com/appcelerator/titanium_mobile/blob/master/android/package.json#L18)

Comment (garymathews) [inline cli/commands/_build/generate.js]: Ah, yes!

**Comment by garymathews** [comment]
@infosia @ewanharris Updated PR

**Comment by ewanharris** [comment]
@garymathews I can't remember what we discussed, did we want to deal with module build in this PR also or split off into a new ticket?

**Comment by garymathews** [comment]
@ewanharris Yes I think we should. This seems like a candidate for `6.1.2`?

**Comment by ewanharris** [comment]
@garymathews Cool, I filed [TIMOB-25031](https://jira.appcelerator.org/browse/TIMOB-25031) for that. I'll get started on an FR for this so that we can take in 6.1.2 if we want. I guess it'll be good to take just to prevent potential footgun moments

**Comment by ewanharris** [comment]
Confirmed issue does not occur for @ssjsamir so merging this PR as I believe it was an issue with my environment

**Comment by infosia** [comment]
Cab we close [TIMOB-24908](https://jira.appcelerator.org/browse/TIMOB-24908) along with this PR?

**Comment by ewanharris** [comment]
It's marked for 6.2.0 so I left it in review until the backport was made and merged

**Pull request merged**